pt/cheatsheet-machine-learning-tips-and-tricks.md

```diff
@@ -1,285 +1,285 @@
 **1. Machine Learning tips and tricks cheatsheet**
 
-&#10230;
+&#10230; Dicas e Truques de Aprendizado de Máquina
 
 <br>
 
 **2. Classification metrics**
 
-&#10230;
+&#10230; Métricas de Classificação
```

Comment:
`classificação` with lowercase, to follow the pattern of all documents

Reply:
Great! I agree.